app/upgrades/vNEXT.go

```diff
@@ -61,7 +114,42 @@ var Upgrade_NEXT = Upgrade{
 		// 3. Update the upgrade handler here accordingly
 		// Ref: https://github.com/pokt-network/poktroll/compare/vPREV..vNEXT
 
+		createZeroBalanceMorseClaimableAccounts := func(ctx context.Context) error {
```

Comment:
Wdyt of moving this into a local non-inline function?

I just feel like it'll be easier to read.